_includes/manuals/1.16/4.4.4_prov_ptables.md

```diff
@@ -120,3 +120,50 @@ cat <<EOF > /tmp/diskpart.cfg
 EOF
 sed '/<\/ntp-client>/ r /tmp/diskpart.cfg' /tmp/profile/autoinst.xml > /tmp/profile/modified.xml
 ```
+##### Configuration 
+
+A partition table entry represents either
+* An explicit layout for the partitions of your hard drive(s). E.G.
+```
+zerombr
+clearpart --all --initlabel
+part /boot --fstype ext3 --size=100 --asprimary
+part /     --fstype ext3 --size=1024 --grow
+part swap  --recommended
+```
+* A script to dynamically calculate the desired sizes. E.G.
+```
+#Dynamic - The below code is to manage the swap size
+
+#get the actual memory installed on the system and divide by 1024 to get it in MB
+usable_ram=$((`awk '$1 ~ /^MemTotal/ {printf "%d\n", $2 / 1024}' /proc/meminfo`))
+
+#check if the memory is less than 2GB then swap is double the memory else it is maximum 24 G for really inactive stuff.
+if [ "$usable_ram" -le 2048 ]; then
+  swap_size=$(($usable_ram * 2))
+else
+  swap_size=$(($usable_ram + 2048))
+fi
+if [ $swap_size -gt 24576 ] ; then
+  swap_size=24576
+fi
+
+#copy all the HDD partitions to the temp file for execution
+cat << EOF > /tmp/diskpart.cfg
+zerombr
+clearpart --all --initlabel
+part swap --size 250 --maxsize "$swap_size" --grow
+part /boot --fstype ext3 --size 100 --asprimary
+part / --fstype ext3 --size 8192 --maxsize 12288 --grow
+part /tmp2 --size 250 --fstype ext3 --grow
+EOF
+```
+The inclusion of the keyword string #Dynamic at the start of a line lets Foreman know that this is not an explicit 
+disk layout and must treated as a shell script, executed prior to the install process and that the explicit partition 
+table will be found at /tmp/diskpart.cfg during the build process.
```

Comment:
I'd enclose `#Dynamic` and `/tmp/diskpart.cfg` in that sentence with backticks, 👍 otherwise